.gitignore

```diff
@@ -21,3 +21,5 @@ sblm.sublime-workspace
 
 /index.js
 .idea
+
+translations
```

Comment:
I thought we need the folder to be there so files can be stored? (otherwise file can't be saved?)

Reply:
Right. It should be the json files only [`a9a206e` (#14)](https://github.com/anyTV/anytv-i18n/pull/14/commits/a9a206e08169259d0b4faf95e35f8bd3c2831e92)